src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -912,27 +908,51 @@ public String getInstructorFeedbackSessionActions(FeedbackSessionAttributes sess
             "data-courseid=\"" + session.courseId + "\" data-fsname=\"" + session.feedbackSessionName + "\" " +
             "data-toggle=\"modal\" data-target=\"#remindModal\">Remind particular students</a></li></ul></div> "
         );
-        
+        String unpublishLinkStyle = "class=\"btn btn-default btn-xs btn-tm-actions session-unpublish-for-test\"";
+        String publishLinkStyle = "class=\"btn btn-default btn-xs btn-tm-actions session-publish-for-test";
+        result.append(getInstructorFeedbackSessionPublishAndUnpublishAction(session, isHome, instructor, publishLinkStyle, unpublishLinkStyle));
+
+        return result.toString();
+    }
+    
+    /**
+     * Returns the link of publish and unpublish action for the session feedback
+     * @param session
+     *         The session details
+     * @param isHome
+     *         Flag whether the link is to be put at homepage (to determine the redirect link in delete / publish)
+     * @param instructor
+     *         The instructor attributes of the session feedback
+     * @param publishLinkStyle
+     *         A string containing the CSS styles of the publish link
+     * @param unpublishLinkStyle
+     *         A string containing the CSS styles of the unpublish link
+     * @return
+     */
+    public String getInstructorFeedbackSessionPublishAndUnpublishAction(FeedbackSessionAttributes session, boolean isHome, InstructorAttributes instructor, String publishLinkStyle, String unpublishLinkStyle) {
```

Comment:
Since the button is displayed differently on different pages, maybe implement another function/override the function in the PageData class for the Results Page?